config/production.json

```diff
@@ -129,7 +129,7 @@
 		"ssr/sample-log-cache-misses": true,
 		"support-user": true,
 		"tools/migrate": true,
-		"2fa/keys-support": false,
+		"2fa/keys-support": true,
```

Comment:
This is now true in all environments.  Any reason to not pull out the flag altogether? The only usage looks to be here:

https://github.com/Automattic/wp-calypso/blob/de18db59f34af488ff47092e28b596ad9dade68a/client/me/two-step/index.jsx#L178

Reply:
Makes sense, I update the config flag in 8192d3e and https://github.com/Automattic/wp-calypso/pull/42293/commits/11b5321083376c953719158df9f3392728d1eec5